app/endpoint/visitor-message/ValidateVisitorMessageEndpoint.ts

```diff
@@ -0,0 +1,29 @@
+import {validate} from '../../lib/validatejs/0_13_1/validate';
+
+const contactUuidFormat = `[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}`;
+
+export default function validateRequest(query: any): any {
+
+    const constraints = {
+        'contactUuid': {
+            presence: {
+                allowEmpty: false,
+            },
+            type: 'string',
+            format: {
+                pattern: contactUuidFormat,
+            },
+        },
+        'msg': {
+            presence: {
+                allowEmpty: true,
```

Comment:
Shouldn't this be false?

Reply:
In the future, when attachments support is added, this field can be empty because the visitor can only send the attachment